src/storybook/decorators.tsx

```diff
@@ -21,3 +22,14 @@ export const Apollo = (DecoratedStory: () => StoryFnReactReturnType): JSX.Elemen
     <Frame style={{ display: 'inline-block' }}>{DecoratedStory()}</Frame>
   </ApolloProvider>
 )
+
+export const Form = (DecoratedStory: () => StoryFnReactReturnType): JSX.Element => {
+  const methods = useForm()
+  return (
+    <Frame style={{ maxWidth: '50rem' }}>
```

Comment:
why you limit the width here?

Reply:
came from the component when i copied the container over, i'll remove it

Reply:
```suggestion
    <Frame>
```